<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MPStop</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="Cinema" name="cinema" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>601</width>
         <height>341</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPlayBack">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>360</y>
         <width>98</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPlayFwd">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>360</y>
         <width>98</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="edtCurrentFrame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPause">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>360</y>
         <width>98</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>&quot;</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="TimeLine" name="timeLineView">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>138</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Input</string>
       </property>
       <widget class="QPushButton" name="btnOpen">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Output</string>
       </property>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>113</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>113</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>113</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Render</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TimeLine</class>
   <extends>QGraphicsView</extends>
   <header>timeline.h</header>
  </customwidget>
  <customwidget>
   <class>Cinema</class>
   <extends>QWidget</extends>
   <header>cinema.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
